studio/lib/helpers.ts

```diff
@@ -266,3 +266,17 @@ export const isValidHttpUrl = (value: string) => {
   }
   return url.protocol === 'http:' || url.protocol === 'https:'
 }
+
+/**
+ * Helper function to remove comments from SQL.
+ * Disclaimer: Doesn't work as intended for nested comments.
```

Comment:
@kishanhitk do you have an example of nested comments that fails?

Reply:
@gregnr 
Yeah, sure.

Input: `/* Outer comment /* Inner comment */ End of outer comment with DROP */`
Output: `End of outer comment with DROP */`

I think we can remove comments recursively to make this work for nested comments like this. But even that won't be completely foolproof, and recursion can have its own overhead. Not worth those risks considering the current use case.

Reply:
Cool thanks for the example. Yes, given that the consequences are low to have a false positive here, I'm okay to merge as is (also assuming nested comments are relatively rare).